src/Components/Login/LoginContainer.jsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import {getloginUserThunkCreator} from "../../redux/auth-reducer";
+import Login from "./Login";
+import {connect} from "react-redux";
+
+
+class LoginContainer extends React.Component {
+
+    componentDidMount() {
+        this.props.getloginUserThunkCreator();
+        //
+        // LoginAPI.postLogin()
+        //     .then(response => {
+        //             if (response.data.accessToken !== "") {
+        //                 let {accessToken, accessTokenExpiresIn, refreshToken, refreshTokenExpiresIn} = response.data;
+        //
+        //                 this.props.setAuthUserData(accessToken, accessTokenExpiresIn, refreshToken, refreshTokenExpiresIn)
+        //             }
+        //
+        //         }
+        //     );
+
+    }
+
+    render() {
+        return (
+            <Login/>
+        )
+    }
+
+}
+
+let mapStateToProps = (state) => ({
+    isAuth: state.auth.isAuth,
+    login: state.auth.login
+
+});
+
+let mapDispatchToProps = (dispatch) => {
+    return {
```

Comment:
I feel here is smth wrong)
Maybe smth like this will be better:
```
let mapDispatchToProps = (dispatch) => {
    return {
        getloginUser: (login, password) => dispatch(getloginUserThunkCreator(login, password))
    }
};
```
But in another comment, I wrote an example of how to do that with the hook.